src/main/java/com/google/flourbot/api/DriveCloudSheet.java

```diff
@@ -12,49 +12,66 @@
 import java.util.List;
 
 public class DriveCloudSheet implements CloudSheet {
-  Sheets sheetsService;
-  String spreadsheetId;
+  private String valueInputOption = "USER_ENTERED";
+  private String insertDataOption = "INSERT_ROWS";
+  private Sheets sheetsService;
+  private String spreadsheetId;
 
   public DriveCloudSheet(String documentId, Sheets documentService) {
     this.spreadsheetId = documentId;
     this.sheetsService = documentService;
   }
 
-  // TODO: replace hardcoded range with calculated range (so far values must be length 3)
-  public void appendRow(List<String> values) throws IOException, GeneralSecurityException {
+  private String getRange(int numEntryColumns, int numSkipColumns) {
+    // Returns range for sheets request by calculating the alphabetic name of
+    // the start and end column based on the number of columns filled and pre-skipped
+
+    String startColumn = this.toAlphabetic(numSkipColumns);
+    String endColumn = this.toAlphabetic(numSkipColumns + numEntryColumns - 1);
+    return "Sheet1!" + startColumn + "1:" + endColumn + "1";
```

Comment:
This looks like the job for String.format(), here you have an example and some context: https://dzone.com/articles/java-string-format-examples

Reply:
I like this so much better!